environments/demo/demo-platform-hmcts-net.yml

```diff
@@ -11,6 +11,14 @@ A:
     ttl: 300
     record:
       - "20.108.232.218"
+  - name: tax-tribunals
+    record:
+     - 51.145.96.41
+     ttl: 300
```

Comment:
ttl indent is off, prob best to move it to the same location as the other record, same issue below